src/datasets/active_profiles.md

```diff
@@ -0,0 +1,140 @@
+- `max_day`: last day in the window the client was seen; the highest value this can be is
+`submission_date`
+- `recency`: age of client in days
+- `frequency`: number of days in the window that a client has returned to use the browser
+after `min_day`
+- `num_opportunities`: given a first appearance at `min_day`, what is the highest number of
+days a client could have
+returned. That is, what is the highest possible value for `frequency`
+
+Since the model is only using these 2 coarse-grained statistics, these columns should make it
+relatively straightforward to interpret why the model made the predictions that it did for a given
+profile.
+
+
+## Latent quantities
+
+The model estimates the expected value for 2 related latent probability variables for a user. The
+values in `prob_daily_leave` give our expectation of the probability that they will become inactive
+on a given day, and `prob_daily_usage` represents the probability that a user will return on a given
+day, *given that they are still active*.
+
+These quantities could be useful for disentangling usage *rate* from the likelihood that a user is
+still using the browser. We could, for example, identify intense users who are at risk of
+churning, or users who at first glance appear to have churned, but are actually just infrequent
+users.
+
+`prob_active` is the expected value of the probability that a user is still active on
+`submission_date`, given their most recent 90 days' of activity. 'Inactive' in this sense
+means that the profile will not use the browser again, whether because they have uninstalled
+the browser or for some other reason.
+
+
+## Predictions
+
+There are several columns of the form `e_total_days_in_next_7_days`, which give the expected
+number of times that a user will show up in the next 7 days (or 14, 21, 28 days). These
+predictions take into account both the likelihood that a user will become inactive in the
+future, as well as their daily propensity to use the browser, given that they are still active.
+The values in `e_total_days_in_next_7_days` will be between 0 and 7.
+
+An estimate for the probability that a client will contribute to MAU is available in the
+column `prob_mau`. This is simply the probability that the user will return at any point in
+the following 28 days, thereby contributing to MAU. Since it is a probability, the values will
+range between 0 and 1, just like `prob_daily_leave` and `prob_daily_usage`.
+
+
+## Attributes
+There are several columns that contain attributes of the client, like `os`, `locale`,
+`normalized_channel`, `normalized_os_version`, and `country`. `sample_id` is also included,
+which can be useful for quicker queries, as the table is clustered by this column in BigQuery.
+
+
+## Remarks on the model
+A way to think about the model that infers these quantities is to imagine a simple process
+where each client is given 2 coins when they become users, and that they flip each day.
```

Comment:
are these fair coins, or are you weighting them somehow? it sounds like not, but want to check